persistence-modules/jimmer/pom.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>jpa-hibernate-cascade-type</artifactId>
+  <properties>
+    <jimmer.version>0.9.37</jimmer.version>
+  </properties>
+
+  <parent>
+    <groupId>com.baeldung</groupId>
+    <artifactId>persistence-modules</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+  </parent>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-dependencies</artifactId>
+        <version>3.4.1</version>
```

Comment:
Where possible, use a version number from the parent. If you must introduce new version numbers, extract properties for them. Same elsewhere in this file.